.github/workflows/main.yml

```diff
@@ -38,7 +38,8 @@ jobs:
           python-version: '3.8'
       - run: sudo apt update
       - run: sudo apt install --no-install-recommends -y x11-xserver-utils
-      - run: pip3 install mypy==0.910 flake8==3.9.2 yapf==0.31.0 --user
+      - run: pip3 install mypy==0.971 flake8==5.0.4 pyright==1.1.271 yapf==0.31.0 --user
       - run: echo "$HOME/.local/bin" >> $GITHUB_PATH
-      - run: mypy -p plugin
+      # - run: mypy -p plugin
```

Comment:
I would remove it, or leave a comment above the line saying why it was disabled.

Reply:
I would expect that we could re-enable it soon so adding a comment for now